rl_studio/README.md

```diff
@@ -12,18 +12,22 @@ To run RL-Studio, first go to dir
 cd ~/PATH/TO/RL-Studio/rl-studio
 ```
 
-and then just typing:
+and then just type (depending on how the dependencies are managed):
 
 ```bash
-python main_rlstudio.py -n [algorithm] -a [agent] -e [environment] -f config/config.yaml
+poetry run python main_rlstudio.py -n [algorithm] -a [agent] -e [environment] -f config/config.yaml # if using Poetry for dependencies
+python main_rlstudio.py -n [algorithm] -a [agent] -e [environment] -f config/config.yaml # if using PIP for dependencies
 ```
 
-where config.yaml contains all project hyperparams and configuration needed to execute correctly.
+if using poetry.
```

Comment:
What is indicating this sentence?

Reply:
That's a typo! Thanks for pointing it out